frontend/src/scenes/trends/PersonModal.tsx

```diff
@@ -87,7 +87,7 @@ export function PersonModal({ visible, view, filters, onSaveCohort }: Props): JS
             footer={
                 <Row style={{ justifyContent: 'space-between', alignItems: 'center', padding: '6px 0px' }}>
                     <Row style={{ alignItems: 'center' }}>
-                        {people && people.count > 0 && (
+                        {people && people.count > 0 && (view === ViewType.TRENDS || view === ViewType.STICKINESS) && (
```

Comment:
Can we combine some of this logic with what we have going on in `showSaveCohortButton`?

Reply:
done